WooCommerce/WooCommerceTests/POS/ViewModels/PointOfSaleDashboardViewModelTests.swift

```diff
@@ -12,7 +12,7 @@ final class PointOfSaleDashboardViewModelTests: XCTestCase {
     override func setUp() {
         super.setUp()
         cardPresentPaymentService = CardPresentPaymentPreviewService()
-        sut = PointOfSaleDashboardViewModel(items: [],
+        sut = PointOfSaleDashboardViewModel(itemProvider: POSItemProviderPreview(),
```

Comment:
We should make a mock object here – even though this is convenient, things should have one job.

Reply:
Right! I've created a `MockPOSItemProvider` on a previous commit but I realized this was sitting in `Yosemite` and testing the mock, rather than the implementation. I've moved it to `WooCommerce` here: https://github.com/woocommerce/woocommerce-ios/pull/13116/commits/3b51e7ab938865fd7537d7e901bcdc7f3d3fd466 as an extension for now, will extract it as soon as we need to add further tests.